Modules/Devices/test/Devices.Application.Tests/Tests/Tiers/Commands/DeleteTier/HandlerTests.cs

```diff
@@ -58,6 +59,26 @@ public async Task Cannot_delete_tier_if_CanBeDeleted_returns_an_error()
         A.CallTo(() => _tiersRepository.Remove(tier)).MustNotHaveHappened();
     }
 
+    [Fact]
+    public void Cannot_delete_queued_for_deletion_tier()
+    {
+        // Arrange
+
+        A.CallTo(() => _tiersRepository.FindById(Tier.QUEUED_FOR_DELETION.Id, A<CancellationToken>._)).Returns(Task.FromResult(Tier.QUEUED_FOR_DELETION));
+        A.CallTo(() => _tiersRepository.GetNumberOfClientsWithDefaultTier(Tier.QUEUED_FOR_DELETION, A<CancellationToken>._)).Returns(Task.FromResult(0));
+        A.CallTo(() => _tiersRepository.GetNumberOfIdentitiesAssignedToTier(Tier.QUEUED_FOR_DELETION, A<CancellationToken>._)).Returns(Task.FromResult(0));
+
+        // Act
+        var acting = async () => await _handler.Handle(new DeleteTierCommand(Tier.QUEUED_FOR_DELETION.Id), CancellationToken.None);
+
+        // Assert
+        var exception = acting.Should().AwaitThrowAsync<DomainException>().Which;
+        exception.Message.Should().Be("The 'Queued for Deletion' Tier cannot be deleted.");
```

Comment:
Testing the error message leads to unnecessary false positives. We should only do this if it is actually necessary. And since you already check the error code, it is unnecessary.